<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_AdjustTransparency</class>
 <widget class="QMainWindow" name="UI_AdjustTransparency">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>调节透明度</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
